app/main.py

```diff
@@ -1,6 +1,29 @@
+import json
+import os
+
+import requests
+
+
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+API_KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    result = requests.get(
+        URL
+        + f"?q={FILTERING}&key={API_KEY}"
+    )
```

Comment:
u can use `params` as param for get instead of f-string 
try to avoid using f-stings in such cases if it possible